src/dosbox.cpp

```diff
@@ -446,15 +446,22 @@ void DOSBOX_Init(void) {
 	Pint->SetMinMax(0,10);
 	Pint->Set_help("How many frames DOSBox skips before drawing one.");
 
-	Pbool = secprop->Add_bool("aspect",Property::Changeable::Always,false);
-	Pbool->Set_help("Do aspect correction, if your output method doesn't support scaling this can slow things down!");
+	Pbool = secprop->Add_bool("aspect", Property::Changeable::Always, true);
+	Pbool->Set_help("Do aspect correction. Affects display modes using non-square\n"
+	                "pixels (such as 320x200 or 640x400), scaling them to maintain\n"
+	                "4:3 aspect ratio the most games were designed for.\n"
+	                "Change to false to use square pixels in all modes.");
 
-	Pmulti = secprop->Add_multi("scaler",Property::Changeable::Always," ");
+	Pmulti = secprop->Add_multi("scaler", Property::Changeable::Always, " ");
 	Pmulti->SetValue("normal2x");
-	Pmulti->Set_help("Scaler used to enlarge/enhance low resolution modes. If 'forced' is appended,\n"
-	                 "then the scaler will be used even if the result might not be desired.\n"
-					 "To fit a scaler in the resolution used at full screen may require a border or side bars,\n"
-					 "to fill the screen entirely, depending on your hardware, a different scaler/fullresolution might work.");
+	Pmulti->Set_help("Scaler used to enlarge/enhance low resolution modes.\n"
+	                 "If 'forced' is appended, then the scaler will be used even if\n"
+	                 "the result might not be desired.\n"
+	                 "To fit a scaler in the resolution used at full screen may\n"
+	                 "require a border or side bars, to fill the screen entirely,\n"
+	                 "depending on your hardware, a different scaler/fullresolution\n"
+	                 "might work.");
```

Comment:
Reworded the last sentence:
Note that some scalers may use black borders to fit the image within your configured display resolution.  If this is undesirable, try either a different scaler or enabling fullresolution output.

Reply:
Replaced the last sentence, thanks! :)